### ISSUE TITLE
Support UDP Sink besides RTSP

### ISSUE THREAD

**Issue by otaviobp** [intel/camera-streaming-daemon#23]

**Comment by anselmolsm** [comment]
Depends on #12

**Comment by otaviobp** [comment]
In mavlink implementation these two messages will needed to support UDP sink:
 - MAV_CMD_VIDEO_START_STREAMING
 - MAV_CMD_VIDEO_STOP_STREAMING

**Comment by lbegani** [comment]
UDP Streaming is now supported - https://github.com/intel/camera-streaming-daemon/pull/98. Support for mavlink messages for video streaming can be taken up in another issue.
